components/checkout/CheckoutSidebarView/CheckoutSidebarView.tsx

```diff
@@ -15,7 +15,7 @@ import { useCheckoutContext } from '../context'
 const CheckoutSidebarView: FC = () => {
   const [loadingSubmit, setLoadingSubmit] = useState(false)
   const { setSidebarView, closeSidebar } = useUI()
-  const { data: cartData, revalidate: refreshCart } = useCart()
+  const { data: cartData, mutate: refreshCart } = useCart()
```

Comment:
I was looking at swr docs but just wanted to reconfirm that this is the correct equivalent

Reply:
<img width="917" alt="Screen Shot 2022-01-13 at 15 29 46" src="https://user-images.githubusercontent.com/1401559/149348795-15289c9c-34fa-4f37-9346-58c2e78576f6.png">
https://swr.vercel.app/blog/swr-v1#change-revalidate-to-mutate